src/main/kotlin/gay/j10a1n15/customscoreboard/utils/Utils.kt

```diff
@@ -29,39 +27,4 @@ object Utils {
         if (index == -1 || index + skip >= size) return null
         return elementAt(index + skip)
     }
-
-    // https://stackoverflow.com/a/74231671/15031174
-    fun Int.ordinal() = "$this" + when {
```

Comment:
I think the new ordinal returns the ending and not the number + the ending, tho i might be wrong